src/main/java/com/fasterxml/jackson/databind/ser/BeanSerializer.java

```diff
@@ -147,7 +147,6 @@ protected BeanSerializerBase asArraySerializer()
          * - have per-property filters
          */
         if ((_objectIdWriter == null)
-                && (_anyGetterWriter == null)
```

Comment:
Why was this check removed? Is it now possible to have meaningful As-Array serialization for POJOs that include any getter?